test-utils/src/main/java/com/ibm/ws/lars/testutils/fixtures/LooseFileRepositoryFixture.java

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ * Copyright (c) 2015 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.ibm.ws.lars.testutils.fixtures;
+
+import java.io.File;
+import java.util.Collection;
+
+import com.ibm.ws.lars.testutils.clients.LooseFileWriteableClient;
+import com.ibm.ws.repository.connections.LooseFileRepositoryConnection;
+import com.ibm.ws.repository.connections.RepositoryConnection;
+import com.ibm.ws.repository.transport.client.RepositoryReadableClient;
+import com.ibm.ws.repository.transport.client.RepositoryWriteableClient;
+
+/**
+ *
+ */
+public class LooseFileRepositoryFixture extends RepositoryFixture {
+
+    private final Collection<String> _assets;
+
+    public static LooseFileRepositoryFixture createFixture(Collection<String> assets) {
```

Comment:
Not sure that we should be passing in this collection of `String` here. It's always going to be empty, right?

If it's not empty, we're probably going to delete whatever files it is pointing to.

Reply:
Changed so that the fixture creates the backing collection rather than having it passed.